src/collection-browser.ts

```diff
@@ -2385,6 +2400,19 @@ export class CollectionBrowser
           --leftColumnPaddingRight: 2.5rem;
         }
 
+        #facet-top-view {
+          --facetTopViewHeight: 200px;
+          --facetTopViewWidth: 180px;
```

Comment:
suggestion: we need to soften this css to remove the JS property.
ideally, this slot will just grow in height when it needs to. perhaps use flex on this? the only requirement is that the slot must show the child's full height. thanks!

also, I see that all of this styling is actually for the demo div, please do remove.